### DISCUSSION TITLE
API routes response always `text/html` on Lambda, instead of `application/json`

### DISCUSSION THREAD

**Discussion by TheDutchCoder** [nuxt/nuxt#16863] [Questions] [answered]
I don't really know whats going on, but when I deploy our app to a Lambda, all the responses come back as text, instead of json on the API routes.

Is this a known issue, or is something wrongly configure in the Lambda or API Gateway?

**Comment by danielroe** [comment]
Do you have serveStatic enabled?

**Reply by TheDutchCoder** [reply]
No it's SSR, but I have a feeling it's actually the gateway that's overwriting the headers.
Let me investigate that and come back with results!

**Reply by TheDutchCoder** [reply]
Yup, that was it, the gateway was configured to always respond 200 with text/html.
Solved!

**Comment by TheDutchCoder** [comment, marked as answer]
For anyone looking for the same thing: our API Gateway was overwriting header responses to always return 200 with text/html.
Removing that made it work!
